### PR TITLE
OBJ-171 refactoring email jurisdiction switch

### PR DESCRIPTION
**Pull request by markpit**
Changing the switch to determine jurisdiction for email - default is cardiff

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java [hunk 1]
```diff
@@ -113,20 +113,13 @@ private EmailContent constructEmailContent(EmailType emailType,
         data.put("to", email);
         data.put("company_name", companyName);
         data.put("company_number", objection.getCompanyNumber());
-        data.put("objection_id", objection.getId());
```
Comment (mwestacott) [inline src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java:116]: Is this not needed?

Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java:116]: It's a dupe.

Comment (markpit) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java:116]: view lines above, it was in twice, intellj highlighted it

Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java:116]: dupe - duplicate

**Pull request merged**